source/Nevermore/Advanced/Concurrency/LockOnlyConcurrencyHandler.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Nito.AsyncEx;
+
+namespace Nevermore.Advanced.Concurrency
+{
+    public class LockOnlyConcurrencyHandler : ITransactionConcurrencyHandler
```

Comment:
Given this is a public library and this an internal implementation detail a sensible default here would be `private`, otherwise this will default to forming part of the public contract.

Reply:
Good call, fixed!